subsys/mgmt/ec_host_cmd/Kconfig.logging

```diff
@@ -0,0 +1,20 @@
+# Host command logging configs
+
+# Copyright (c) 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config EC_HOST_CMD_LOG_DEFAULT
+	int "Default logging Host Command level"
+	default 1
+	help
+	  Default Host Command logging level. The value should equals one of the enum
+	  in the enum ec_host_cmd_log_level.
+
+config EC_HOST_CMD_LOG_BUF_MAX
+	int "Max buffer to store request or response"
+	default 129
+	help
+	  Set maximum size of buffer to store request/response in hex. The amount of
```

Comment:
"in bytes"? maybe it's obvious unsure...

Reply:
Done.